pkg/controller/migmigration/task.go

```diff
@@ -155,8 +158,8 @@ var StageItinerary = Itinerary{
 		{Name: WaitForStaleStagePodsTerminated, Step: StepPrepare},
 		{Name: CreateRegistries, Step: StepPrepare, all: IndirectImage | EnableImage | HasISs},
 		{Name: CreateDirectImageMigration, Step: StepStageBackup, all: DirectImage | EnableImage},
-		{Name: QuiesceApplications, Step: StepStageBackup, all: Quiesce},
-		{Name: EnsureQuiesced, Step: StepStageBackup, all: Quiesce},
+		{Name: QuiesceApplications, Step: StepStageBackup, any: Quiesce},
+		{Name: EnsureQuiesced, Step: StepStageBackup, any: Quiesce},
 		{Name: CreateDirectVolumeMigration, Step: StepStageBackup, all: DirectVolume | EnableVolume},
```

Comment:
I'm wondering why this was changed. The two should be equivalent if there's only one condition -- i.e. "any of this one thing must be true" and "all of this one thing must be true" are functionally equivalent. Currently all single-condition checks use "all".

Reply:
@sseago fixed